swift-generator/src/main/java/com/facebook/swift/generator/template/StructContext.java

```diff
@@ -42,6 +42,7 @@ public void addField(final FieldContext field)
         this.fields.add(field);
     }
 
+    @Deprecated
```

Comment:
I don't know how the swift project handles removal of publicly exposed methods, but other OSS projects usually deprecate methods for one or two releases and then remove them.